src/plugins/analysis/hash/code/hash.py

```diff
@@ -17,7 +17,8 @@ class AnalysisPlugin(AnalysisBasePlugin):
     FILE = __file__
 
     def additional_setup(self):
-        self.hashes_to_create = self._get_hash_list_from_config(self.config)
+        hashes = getattr(cfg, self.NAME, {}).get('hashes', 'sha256')
+        self.hashes_to_create = _parse_comma_separated_list(hashes)
```

Comment:
My reasoning was that it ideally should not be used from outside. But since I don't see a way to replace its usage without the change to toml so yeah, I'll make it public.